src/components/HomeSponsors.vue

```diff
@@ -5,9 +5,9 @@
         <i class="fa fa-heart" aria-hidden="true"></i>
         Nos sponsors
       </h2>
-      <b-row>
-        <b-col sm="6" md="4" lg="3" v-for="sponsor of sponsors" :key="sponsor.name">
-          <a class="HomeSponsors__item" :href="sponsor.href">
+      <b-row class="HomeSponsors__row">
+        <b-col class="HomeSponsors__item" sm="6" md="4" lg="3" v-for="sponsor of sponsors" :key="sponsor.name">
+          <a target="_blank" :href="sponsor.href">
```

Comment:
can we add the following attribute to improve SEO : `rel="noopener noreferrer"` ?

Reply:
Ok I will add the title too :o